openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -4,7 +4,11 @@
 
 from opaque_keys.edx.keys import CourseKey
 
-from openedx_events.enterprise import signals  # pylint: disable=unused-import
+# Each new folder with signals must be manually imported in order for the signals to be cached
+#   and used in the unit tests. Using 'disable=reimported' with pylint will work,
+#   because we just use the cached signal list.
+from openedx_events.content_authoring import signals  # pylint: disable=unused-import
+from openedx_events.enterprise import signals  # pylint: disable=reimported
```

Comment:
[suggestion] Maybe add something like the following here, and especially to the `learning` import, to help someone find the relevant comment?
```suggestion
from openedx_events.enterprise import signals  # See note above; pylint: disable=reimported
```

Reply:
Fixed (on the learning import)

Reply:
added to the learning import